packages/es-components/src/components/containers/modal/Modal.js

```diff
@@ -317,7 +321,7 @@ Modal.propTypes = {
   ])
 };
 
-Modal.defaultProps = {
+export const defaultProps = {
```

Comment:
Is there a benefit to exporting this?

Reply:
Yes. Typescript has much better processing on JS files when you export an object as opposed to just setting a static variable. It tends to view anything "internal" as cruft that it can throw away in the type declarations, but anything exported will be fully processed.

Reply:
So something like `Modal.defaultProps` will either get set as `any`, or will be absent from the final type declaration altogether.

Reply:
Thanks for the explanation!